src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -60,4 +60,20 @@ $toggle-switch-circle-size: $toggle-switch-height - 2 * $toggle-switch-spacing;
       transition: $toggle-switch-transition;
     }
   }
+
+  &__error-message {
+    display: none;
+    
+    &-active {
+      display: block;
+      font-family: Source Sans Pro;
```

Comment:
pls use sass mixin `@mixin text `from `src/components/00-materials/styles/typography.scss` and remove the most css attributes from here.